codegen/smithy-python-codegen/src/main/java/software/amazon/smithy/python/codegen/SetupGenerator.java

```diff
@@ -111,6 +111,7 @@ private static void writePyproject(
                     reportUnusedFunction = false
                     reportUnusedVariable = false
                     reportUnnecessaryComparison = false
+                    reportUnusedClass = false
```

Comment:
This is needed because union deserializers are classes, event stream deserializers are never called as they are filtered out currently.